be/src/vec/data_types/serde/data_type_array_serde.cpp

```diff
@@ -220,19 +221,19 @@ Status DataTypeArraySerDe::serialize_one_cell_to_hive_text(
 
 void DataTypeArraySerDe::write_one_cell_to_jsonb(const IColumn& column, JsonbWriter& result,
                                                  Arena* mem_pool, int32_t col_id,
-                                                 int row_num) const {
-    result.writeKey(col_id);
+                                                 int64_t row_num) const {
+    result.writeKey(cast_set<JsonbKeyValue::keyid_type>(col_id));
```

Comment:
add a comment  for the cast_set, seems `JsonbKeyValue::keyid_type` is uint16_t

Reply:
done